### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Colter-Garrison**

### CODE REVIEW

File: __tests__/animals.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const pool = require('../lib/utils/pool');
```
Comment (julienisbet) [inline __tests__/animals.test.js:1]: tests look great!

File: __tests__/animals.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('backend-express-template routes', () => {
```
Comment (julienisbet) [inline __tests__/animals.test.js:6]: don't forget to update your test description - this gets output when you run the tests so it help figure out where things are breaking

**Review by julienisbet — APPROVED** [review]
awesome work @Colter-Garrison! (I'm helping Forest with some grading this mornign). I have almost no comments -- your commit history looks amazing and all your tests are really thorough. Nice work!

File: __tests__/animals.test.js (hunk 2 above)
Comment (Colter-Garrison) [reply __tests__/animals.test.js:6]: Good to know! Thanks!

**Pull request merged**